### PR TITLE
Update antivirus-microsoft-defender-settings-windows.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated AllowIOAVProtection

Fixes#https://github.com/MicrosoftDocs/memdocs/issues/3475

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

**Comment by traya1** [comment]
Learn Build status updates of commit _[00f2a7f](https://github.com/msbemba/memdocs/commits/00f2a7fc3f7d0bcc06d81456eef28b62340a2d5d)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/intune/protect/antivirus-microsoft-defender-settings-windows.md](https://github.com/msbemba/memdocs/blob/patch-187/memdocs/intune/protect/antivirus-microsoft-defender-settings-windows.md) | :warning:Warning | | [Details](#user-content-d0c2f0961f3c12b7f3897340cca4b7a5-330120b8)

<a id="d0c2f0961f3c12b7f3897340cca4b7a5-330120b8"></a>
### [memdocs/intune/protect/antivirus-microsoft-defender-settings-windows.md](https://github.com/msbemba/memdocs/blob/patch-187/memdocs/intune/protect/antivirus-microsoft-defender-settings-windows.md)
 - **Line 156, Column 8**: **[Warning: file-not-found - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: 'windows/client-management/mdm/policy-csp-defender'.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202301101915396218-3498/BuildReport?accessString=8ddac534b4134d6c625eedcff756c65da71bf89c09e62b8bdd91bb9aa860c7bf).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

File: memdocs/intune/protect/antivirus-microsoft-defender-settings-windows.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ These settings are available in the following profiles:
   - **Yes** - Network protection is turned on. Device users can't change this setting.
 
 - **Scan all downloaded files and attachments**  
-  CSP: [EnableNetworkProtection](/windows/client-management/mdm/policy-csp-defender#defender-enablenetworkprotection)
+  CSP: [AllowIOAVProtection](windows/client-management/mdm/policy-csp-defender#allowioavprotection)
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/antivirus-microsoft-defender-settings-windows.md:156]:
```suggestion
  CSP: [AllowIOAVProtection](/windows/client-management/mdm/policy-csp-defender#allowioavprotection)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**